### ISSUE TITLE
Merging Two sets of data. 關於將兩組數據合併

### ISSUE THREAD

**Issue by slui** [dkwingsmt/plugin-wheres-my-fuel-gone#15]
I have two devices and two sets of data (*Even i have the same account). I want to merge it into one, but since there is no upload or output for data sets, how am i able to do it manually? 

因為我有兩個不同的電腦及兩組數據(儘管用同一個帳號),因此我想把兩組數據合併。但是，由於沒有將數據匯出或匯入的選項，我應該如何手動進行?

**Comment by dkwingsmt** [comment]
A record consists of 2 files.

For the "record" file, it is an array of objects,
```
[ { ... }, { ... }, ... { ... } ]
```
So you need to concat these two files, and replace the `] [` with `,`.

For the "bucket" file, it's a temporary file that records the last sortie that each ship participated. It only affects the buckets for now, and if you have repaired all the ships intended, it will not make any difference. Simply use the latest bucket file should be enough.

**Comment by slui** [comment]
I did some testing , but the time for the record are messed up, or to be exactly, its not sorted....
I know each piece of the data is originated from a class, so it is possible, in future, that you would consider adding some functionality  for users to upload or output data and in addition, will be sorted?

Just like akashic records, it has those functionality, and the data are sorted later after new data has been uploaded....

**Comment by dkwingsmt** [comment]
Yeah I should definitely add sorting the data (actually I thought I sorted it although it turns out I didn't).

As for importing and outporting, yeah that could be a thing as well.

**Comment by slui** [comment]
Thanks, looking forward on that update sometime later.....
